.pylintrc

```diff
@@ -69,85 +69,14 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
-disable=print-statement,
-        parameter-unpacking,
-        unpacking-in-except,
-        old-raise-syntax,
-        backtick,
-        long-suffix,
-        old-ne-operator,
-        old-octal-literal,
-        import-star-module-level,
-        non-ascii-bytes-literal,
-        raw-checker-failed,
+disable=raw-checker-failed,
```

Comment:
Good move, it's cleaner if we disable only what we actually need to disable.

Reply:
this was actually forced by new pylint, as they do not support those "disables" anymore (because they dropped python2 support)